target/linux/layerscape/patches-5.4/900-layerscape_fix_support_of_LS1012A-RDB.patch

```diff
@@ -0,0 +1,91 @@
+--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
++++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
+@@ -6,7 +6,8 @@
+  *
+  */
+ /dts-v1/;
+-
```

Comment:
Nitpick: I would have just kept this empty line.

Reply:
Done.